modApiServer/src/org/aion/api/server/rpc3/types/RPCTypesConverter.java

```diff
@@ -0,0 +1,378 @@
+package org.aion.api.server.rpc3.types;
+
+import org.json.JSONObject;
+import static org.aion.api.server.rpc3.types.RPCTypes.*;
+import java.util.regex.Pattern;
+import org.aion.api.server.rpc3.RPCExceptions.ParseErrorRPCException;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import java.math.BigInteger;
+
+/******************************************************************************
+*
+* AUTO-GENERATED SOURCE FILE.  DO NOT EDIT MANUALLY -- YOUR CHANGES WILL
+* BE WIPED OUT WHEN THIS FILE GETS RE-GENERATED OR UPDATED.
+*
+*****************************************************************************/
+public class RPCTypesConverter{
+
+    public static class StringConverter{
+
+        public static String decode(Object s){
+            if(s==null) return null;
+            return s.toString();
+        }
+
+        public static String encode(String s){
+            return s;
+        }
+    }
+
+    public static class LongConverter{
+        private static final Pattern hexPattern = Pattern.compile("^0x[0-9a-fA-F]+");
+        private static final Pattern decPattern = Pattern.compile("^[0-9]+");
```

Comment:
Does the converter support the negative value? If the answer is yes, the decPattern needs to be revised.

Reply:
Yep, it does. My test case picked up that issue.